moveit_core/robot_state/include/moveit/robot_state/robot_state.h

```diff
@@ -1773,6 +1773,10 @@ as the new values that correspond to the group */
   bool has_acceleration_;
   bool has_effort_;
 
+  /** \brief We at least 5 steps per trajectory for testing jump thresholds with computeCartesianPath. Otherwise
+   * testJointSpaceJump doens't work well. */
```

Comment:
Lets not use the 'we' pronoun... "At least 4 steps are used per trajectory for testing jump thresholds..."